Project.toml

```diff
@@ -14,7 +14,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 BinaryProvider = "0.5.9"
-CSDP_jll = "=6.2.0"
+CSDP_jll = "=600.200.1"
```

Comment:
```suggestion
CSDP_jll = "=6.2.0, =600.200.1"
```

Reply:
If things are compatible, then there's no need to force new updates. It makes it hard if people want to install CSDP and Ipopt, for example. (In this case it's okay because I bumped all the COIN-OR packages simultaneously, but this won't always be the case.)